### DISCUSSION TITLE
Audible-like theme and a feature to remember where you left of before going to sleep

### DISCUSSION THREAD

**Discussion by blckbrd14** [TortugaPower/BookPlayer#559] [Ideas]
Great work all around with this app, thank you! Here are two suggestions for new features:

• A theme that mimics Audible app’s dark grey scheme for all the people who have a strong emotional connection to that specific tint from hours of using Audible.
• A weird feature I haven’t seen in any app: a sleeping aid that remembers your original place in the book. In other words, the ability to play books without registering the final position in the timeline. Sometimes you just want to sleep and not pay attention to the book. So you hit this special play button and the book plays for x minutes. When it stops, the location resets to the position you started from.

**Comment by pichfl** [comment]
(edit: I learned that we can move issues to discussions 🎉)

1. **New theme:** New themes will drop more regularly in the future. The suggestion is duly noted. We won't copy verbatim, but I think getting inspiration is acceptable.
2. There is no need for a special play button for this once the **new bookmarks feature** arrives. #127 is tracking this. We will provide automatic bookmarks for the last time you hit play/pause.
